src/lesson6_task1/MilitaryTransport.java

```diff
@@ -0,0 +1,53 @@
+package lesson6_task1;
+
+public class MilitaryTransport extends AirTransport {
+    boolean isCatapult;
+    int countRocket;
+
+    //конструктор
+    public MilitaryTransport(int powerHours, String brand, int maxSpeed, int minRunwayLength,
+                             float weight, float wingspan, int countRocket, boolean isCatapult) {
+        super(powerHours, maxSpeed, brand, weight, wingspan, minRunwayLength);
+        this.countRocket = countRocket;
```

Comment:
@Nadya101  нету входного параметра для countRocket
```suggestion
    public MilitaryTransport(int powerHours, String brand, int maxSpeed, int minRunwayLength,
                             float weight, float wingspan, int countRocket, boolean isCatapult,
                             int countRocket) {
        super(powerHours, maxSpeed, brand, weight, wingspan, minRunwayLength);
        this.countRocket = countRocket;
```